scripts/calendar/hindu_generator.py

```
@@ -0,0 +1,733 @@
+
+YEAR_TEMPLATE = "        {year}: ({date}),"
+
+CALENDARS = {
+    "HINDU": "Hindu",
+}
+
+HINDU_HOLIDAYS = (
+    ("DIWALI_INDIA", lambda y: get_diwali(y)),
+    ("DUSSEHRA", lambda y: get_dussehra(y)),
+    ("HOLI", lambda y: get_holi(y)),
+    ("JANMASTHAMI", lambda y: get_janmashtami(y)),
```

Ruff (0.15.12)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)